cost/azure/databricks/rightsize_compute/azure_databricks_rightsize_compute.pt

```diff
@@ -183,7 +187,6 @@ credentials "auth_flexera" do
   tags "provider=flexera"
 end
 
-
 ###############################################################################
 # Pagination
 ###############################################################################
```

Comment:
Line 204 - `pagination_databricks_events` is declared but is not used. Should this be removed?